handsontable/src/core/hooks/constants.js

```diff
@@ -3204,6 +3222,24 @@ export const REGISTERED_HOOKS = [
    */
   'afterDialogFocus',
 
+  /**
+   * Fired by {@link Loading} plugin after showing the loading indicator. This hook is fired when {@link Options#loading}
+   * option is enabled.
+   *
+   * @since 16.1.0
+   * @event Hooks#afterLoadingShow
+   */
+  'afterLoadingShow',
```

Comment:
The entry is duplicated with https://github.com/handsontable/handsontable/pull/11792/files#diff-e621f5d01d348ff174ab895c46d5df06ccbb3bab84b1b5dba6d5bd78cbe3235dR3195. The same with `afterLoadingHide`.